src/tests/pages/ProductList.test.js

```diff
@@ -0,0 +1,23 @@
+import '@testing-library/jest-dom';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { Furnitures } from '../../Furnitures';
+describe('Testing Products List Page', () => {
+  const setProducts = jest.fn();
+  test('should navigate to Product List Page', () => {
+    render(<Furnitures />);
```

Comment:
This test is wrong, you need to render the app component and then navigate to the products page, this is to test a similar behavior like the users do on the screen but with the testing library.